docs/en/unloading/unload_using_insert_into_files.md

```diff
@@ -63,11 +63,11 @@ INSERT INTO
 FILES(
     "path" = "hdfs://xxx.xx.xxx.xx:9000/unload/data1",
     "format" = "parquet",
+    "compression" = "uncompressed",
```

Comment:
1. target_max_file_size = 1073741824
2. We should set `compression = lz4` in some cases, and use `uncompressed` only when setting `target_max_file_size`,  because currently we haven't a good implementation of the target size on compressed files.